src/js/components/jtable/TableCell.jsx

```diff
@@ -24,34 +37,40 @@ export class TableCell extends Component {
             classNames.push(className);
         }
 
+        const {
+            linkProps,
+            tagOrComponent
+        } = generateLink('div', href, linkTo);
```

Comment:
why do we even have `TableCell`?? why not just build in logic to `TableRow` or styles based on immediate child components? it would be more flexible that way and less code for the end user...